src/resolvers/dockerfile.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2019 Balena Ltd.
+ * Copyright 2020 Balena Ltd.
```

Comment:
I understand that the initial date is the most important one in copyright notices (as it establishes who "invented" the code first, and therefore has a valid claim to copyright). The good options are:
```
Copyright 2019 Balena Ltd.
```
or
```
Copyright 2019-2020 Balena Ltd.
```
When there is a range as in the second option, the latest date is simply a hint as to when the code was last revised. It doesn't mean that copyright stops in year 2020. But importantly, it indicates that it starts in 2019.

Reply:
ohh TIL, I always thought we just bump it. OK, will change